### ISSUE TITLE
Windows | VS 2019 | C99 (C++11) |  32bit support

### ISSUE THREAD

**Issue by mkirchner** [mkirchner/gc#27]
Forking the MSVC discussion from the GCC discussion [here](https://github.com/mkirchner/gc/issues/17).

Since I don't have a windows/MSVC setup there is no "official" MSVC support; I added a "help required" badge to highlight that -- contributions are very welcome!

**Comment by ganesh-k13** [comment]
Hi @mkirchner, I'll look into making it compatible with windows.

**Comment by ssrlive** [comment]
We can add a macro at the beginning of `gc.h`, 
```
#if defined(_MSC_VER)
#define __builtin_frame_address(x)  ((void)(x), _AddressOfReturnAddress())
#endif
```

**Comment by mkirchner** [comment]
Hey @ssrlive , @ganesh-k13, with #38 merged, is this still an issue?

**Comment by ssrlive** [comment]
Not any more

**Comment by ganesh-k13** [comment]
> Can do. Would you be willing/able/have time to to start a new branch and see how to support that using github actions?

Sure @mkirchner, will do.

**Comment by mkirchner** [comment]
Ok, I'll close this as fixed in #38. LMK if we need to re-open.
